### ISSUE TITLE
Multi-select quick pick needs polish

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#163230]
Was reviewing #159310 and spotted a couple of issues:
- Checkboxes appear to be different than our VS Code-specific checkboxes
- Input tag is not vertically centered

<img width="644" alt="CleanShot 2022-10-10 at 15 57 04@2x" src="https://user-images.githubusercontent.com/25163139/194964405-eaa2c128-4140-4dec-952c-a1c79c80415e.png">

<img width="88" alt="CleanShot 2022-10-10 at 16 00 29@2x" src="https://user-images.githubusercontent.com/25163139/194964710-103cde95-705f-478d-91ae-37860b65fcc6.png">

<img width="102" alt="CleanShot 2022-10-10 at 16 00 23@2x" src="https://user-images.githubusercontent.com/25163139/194964695-96726d72-3f6e-40fd-aa3c-d6bd4151921e.png">

## Checkbox example

From the Settings UI..

<img width="272" alt="CleanShot 2022-10-10 at 15 59 43@2x" src="https://user-images.githubusercontent.com/25163139/194964597-7b4ccd9e-cac5-4a2d-a607-fd614ad34efc.png">

**Comment by zhouxinghong** [comment]
![image](https://github.com/microsoft/vscode/assets/54253524/7084f95a-2247-4f9c-809b-5155f93a6fb6)

**Comment by TylerLeonhardt** [comment]
@aeschli said Regardless of the checkboxes used, they should still get their own quickpick.... color ids...

**Comment by daviddossett** [comment]
As in color vars specific to the quick pick?

**Comment by TylerLeonhardt** [comment]
That's my interpretation of what @aeschli said.

**Comment by aeschli** [comment]
Yes, we already have a set of quickpick specific colors: https://code.visualstudio.com/api/references/theme-color#quick-picker-colors

Something like `quickInputList.checkboxBackground`
